Notebook extension broken

Running the `yo code` for the notebook template results in a project that doesn't build or run.

The error is

`Cannot find module 'vscode-notebook-error-overlay'. Did you mean to set the 'moduleResolution' option to 'nodenext', or to add aliases to the 'paths' option?`

--- a/generators/app/templates/ext-notebook-renderer/src/client/tsconfig.json
+++ b/generators/app/templates/ext-notebook-renderer/src/client/tsconfig.json
@@ -5,6 +5,7 @@
     "noEmit": true,
     "rootDir": ".",
     "module": "esnext",
+    "moduleResolution": "bundler",
     "lib": ["ES2019", "dom"],
     "types": ["webpack-env", "vscode-notebook-renderer"],
   }

